evm-contracts/src/v0.6/tests/VRFTestnetD20.sol

```diff
@@ -1,53 +0,0 @@
-pragma solidity 0.6.6;
```

Comment:
It's fine to get rid of this, but I'm surprised that the golang [TestD20ExampleContract](https://github.com/smartcontractkit/chainlink/blob/feature%2F%23175865202-d20-example/core/services/vrf/vrf_testnet_d20_test.go) is still passing.